src/bse202/routes/user/search.py

```diff
@@ -0,0 +1,42 @@
+# user/search.py
+
+from flask import Blueprint, request, jsonify
+from ...db import get_db
+import logging
+
+search_bp = Blueprint("search", __name__, url_prefix="/search")
```

Comment:
```py

from .blueprint import user_blueprint

@user_blueprint.get("/search")
def search_games():
    # function content
```

please look around for more examples, there are other files where this pattern is used